src/index.js

```diff
@@ -81,4 +81,40 @@ function RebillyExperimentalAPI({apiKey = null, sandbox = false, timeout = baseT
     return createExperimentalApiInstance({apiHandler});
 }
 
-export {Errors as RebillyErrors, RebillyExperimentalAPI, cancellation};
+/**
+ * Create an instance of the experimental Rebilly API
+ * @param apiKey {string} private API key; if provided will be used for all requests
+ * @param sandbox {boolean} whether to use the sandbox endpoint or the live
+ * @param version {string} specify a different version of the API to use than the current stable release
+ * @param timeout {number} timeout in milliseconds
+ * @param organizationId {string} Organization identifier in scope of which need to perform request (if not specified, the default organization will be used)
+ * @param urls {object} which urls the sdk will use for the base url for live or sandbox modes
+ * @returns {{histograms, reports, customers, setEndpoints, setTimeout}}
+ * @constructor
+ */
+function RebillyStorefrontAPI({apiKey = null, sandbox = false, version = 'v1', timeout = baseTimeoutMs, organizationId = null, urls = baseEndpoints} = {}) {
```

Comment:
This is inconsistent. Above we camelCase API as `Api`. I prefer `Api` too.
```suggestion
function RebillyStorefrontApi({apiKey = null, sandbox = false, version = 'v1', timeout = baseTimeoutMs, organizationId = null, urls = baseEndpoints} = {}) {
```

Inconsistent camelCasing of API.

Reply:
I see the issue. It's consistent with RebillyAPI and RebillyExperimentalAPI.

Reply:
The `baseTimeoutMs` will cause significant issues because the value is too small. This is an insane default.

Reply:
>Inconsistent camelCasing of API.

It's not though.
https://github.com/Rebilly/rebilly-js-sdk/blob/9e2b2ddae1e39752e2187baf55cf032e91bba7fc/src/index.js#L24
and
https://github.com/Rebilly/rebilly-js-sdk/blob/9e2b2ddae1e39752e2187baf55cf032e91bba7fc/src/index.js#L59
Even in your follow up message you use `RebillyAPI` and not `RebillyApi`.

I can make this a major breaking version and update the naming convention.

I can also make changes to the baseTimeoutMs, what would you recommend as a timeout? This value was last changed 3 years ago as part of the initial commit

<img width="1356" alt="Screen Shot 2020-05-19 at 3 04 54 PM" src="https://user-images.githubusercontent.com/13054039/82367636-24f64780-99e2-11ea-880f-f1f6e3de385c.png">

Reply:
Yes, it is inconsistent with everything except RebillyAPI and RebillyExperimentalAPI (as I commented). I don't get your point.

I believe there was a typo in that initial commit with a 0 missing.

Reply:
We use `RebillyAPI`, `RebillyExperimentalAPI` currently in the SDK. How is using `RebillyStorefrontAPI` not consistent with that?

If you're saying it should be `RebillyStorefrontApi` then I should change `RebillyApi`, `RebillyExperimentalApi` which would causing a breaking change in the SDK.

Reply:
I'll make the change for the default timeout to be 1 minute instead of 6 seconds.

Reply:
Perhaps you misread my comment:

> It's consistent with RebillyAPI and RebillyExperimentalAPI.

> ### consistent

It's inconsistent with the rest of the usages throughout the SDK.

Reply:
Yes, I'm essentially saying to leave it as you have it now.

Reply:
Oh yeah that's the confusion. I'll update elsewhere in the app as the suggestion you left above is the public facing function.